packages/renderer/src/lib/kube/KubernetesDashboard.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+import { Link, NavPage } from '@podman-desktop/ui-svelte';
+
+import KubernetesCurrentContextConnectionBadge from '/@/lib/ui/KubernetesCurrentContextConnectionBadge.svelte';
+import {
+  kubernetesCurrentContextConfigMaps,
+  kubernetesCurrentContextDeployments,
+  kubernetesCurrentContextIngresses,
+  kubernetesCurrentContextNodes,
+  kubernetesCurrentContextPersistentVolumeClaims,
+  kubernetesCurrentContextRoutes,
+  kubernetesCurrentContextSecrets,
+  kubernetesCurrentContextServices,
+  kubernetesCurrentContextState,
+} from '/@/stores/kubernetes-contexts-state';
+import { NO_CURRENT_CONTEXT_ERROR } from '/@api/kubernetes-contexts-states';
+
+import { kubernetesContexts } from '../../stores/kubernetes-contexts';
+import ConfigMapSecretIcon from '../images/ConfigMapSecretIcon.svelte';
+import DeploymentIcon from '../images/DeploymentIcon.svelte';
+import IngressRouteIcon from '../images/IngressRouteIcon.svelte';
+import NodeIcon from '../images/NodeIcon.svelte';
+import PvcIcon from '../images/PVCIcon.svelte';
+import ServiceIcon from '../images/ServiceIcon.svelte';
+import KubernetesDashboardGuideCard from './KubernetesDashboardGuideCard.svelte';
+import KubernetesDashboardResourceCard from './KubernetesDashboardResourceCard.svelte';
+import KubernetesEmptyPage from './KubernetesEmptyPage.svelte';
+
+let noContexts = $derived($kubernetesCurrentContextState.error === NO_CURRENT_CONTEXT_ERROR);
+let currentContextName = $derived($kubernetesContexts.find(context => context.currentContext)?.name);
```

Comment:
suggestion (non-blocking): I know typescript is able to infer currentContextName properly, but as a reader I might not know that it can be undefined

```suggestion
let currentContextName: string | undefined = $derived($kubernetesContexts.find(context => context.currentContext)?.name);
```

Reply:
@axel7083 : if you read inside an IDE it's telling you the type 'string | undefined', so reading it outside of an IDE ?

Reply:
Reading in GitHub, yes the IDE is showing the type